### ISSUE TITLE
question about function get_state?

### ISSUE THREAD

**Issue by hustcxx** [HLTCHKUST/Mem2Seq#30]
in function get_state (),return Variable(torch.zeros(bsz, self.embedding_dim)).cuda(),and then is used in forward function of 
prob   = self.softmax(torch.sum(m_A*u_temp, 2)),isn't the result zero???
